tests/controls/gridtest.cpp

```diff
@@ -659,6 +659,12 @@ TEST_CASE_METHOD(GridTestCase, "Grid::RangeSelect", "[grid]")
     if ( !EnableUITests() )
         return;
 
+#ifdef __WXGTK20__
+    // Works locally, but not when run on Travis CI.
```

Comment:
"Travis CI" is a copy/paste, should be "GitHub CI", right?

Reply:
Oops! will correct this and the others in a moment, thanks.